Sources/fetch_app_status.rb

```diff
@@ -4,28 +4,32 @@
 
 def get_app_state(app)
   
-  version_info = app.get_live_app_store_version
   edit_version_info = app.get_edit_app_store_version
-  appstore_version_info = app.get_app_store_versions.first
+  in_review_version_info = app.get_in_review_app_store_version
+  pending_version_info = app.get_pending_release_app_store_version
+  latest_version_info = app.get_latest_app_store_version
```

Comment:
I didn't know that. `get_in_review_app_store_version`  Thank you!

Reply:
`get_in_review_app_store_version` returns the status when the app store reviewer has started the build review process.